brainstormforce/ultimate-addons-for-gutenberg
# Improvement: Typography - Added Overall Reset functionality in Typography Component.

### Description

- Need a reset for the whole typography control group: https://d.pr/i/H12jkt
Elementor: https://d.pr/i/6Nq9le

### Screenshots

- SS - https://share.getcloudapp.com/12ugzAow

### Types of changes




- Improvement

### How has this been tested?

- Drag and drop any block, Add typography and click on reset.
- Add typography, save the setting and click on reset. The setting should be reset after clicking on the icon.

### Checklist:
